SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Link.h

```diff
@@ -114,10 +114,11 @@ class LinkTraitsContainer;
 
 
 /// Class to hold 0-or-1 pointer. The interface is similar to std::vector (size/[]/begin/end), plus an automatic convertion to one pointer.
-template < class T, class TPtr = T* >
+template < class T, class TDestPtr, class TPtr = T* >
```

Comment:
I would just add a breaking flag and some quick words about this change in the PR description so that we can link it.

Reply:
You are right.